src/charm.py

```diff
@@ -180,13 +194,19 @@ def _on_start_simulation_action(self, event: ActionEvent) -> None:
             event.fail(message=f"Failed to execute simulation: {str(e.stdout)}")
 
     def _generate_ue_config(self) -> str:
+        if self._relation_created(RFSIM_RELATION_NAME):
+            sst = self.rfsim_requirer.sst if self.rfsim_requirer.sst else 1
```

Comment:
Why do we want to default to `1` if SST is missing from the relation data? 
IMO:
1. All the checks we do, should prevent the code from getting to this point if SST is missing
2. Even if by any miracle we get that far with SST missing from the relation data, should we be masking that fact by falling to back to a default value? Missing SST = invalid relation data.

Reply:
Good point, I removed this